src/scripts/logs.ts

```diff
@@ -103,14 +115,27 @@ export default (
           'The filter by which the logs will be filtered (e.g. eventType eq "user.session.start").',
       })
       .option('since', {
-        type: 'string',
         description: 'The start date of the logs to retrieve.',
-        coerce: (date: string) => readonlyDate(date),
+        coerce: (date: string) => coercedDate(date),
       })
       .option('until', {
-        type: 'string',
         description: 'The end date of the logs to retrieve.',
-        coerce: (date: string) => readonlyDate(date),
+        coerce: (date: string) => coercedDate(date),
+      })
+      .option('within', {
+        description:
+          'The start date of the logs to retrieve, expressed as a duration to be applied to now (e.g. 1d/P1D for 1 day, 2w/P2W for two weeks).',
```

Comment:
**suggestion**: Perhaps we could add a link here to e.g., https://en.wikipedia.org/wiki/ISO_8601#Durations, to benefit people unfamiliar with ISO 8601 duration formats?

Reply:
Done.